passkey-api/introduction.mdx

```diff
@@ -14,7 +14,34 @@ If you need a full, **open-source** auth solution, check out [Hanko Auth](/intro
 
 </Tip>
 
-## What it looks like
+<CardGroup cols={2}>
+  <Card
+    title="Next.js with Auth.js"
+    icon={
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        className="icon icon-tabler icon-tabler-brand-nextjs"
+        width="30"
+        height="30"
+        viewBox="0 0 30 30"
+        strokeWidth="2"
+        stroke="currentColor"
+        fill="none"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      >
+        <path stroke="none" d="M0 0h24v24H0z" fill="none"></path>
+        <path d="M9 15v-6l7.745 10.65a9 9 0 1 1 2.255 -1.993"></path>
+        <path d="M15 12v-3"></path>
+      </svg>
+    }
+    href="/quickstarts/fullstack/next"
```

Comment:
This card redirects to the next guide right? Why so?

Reply:
...because I copied it from /instruction.mdx :P

Thanks! Fixed